cfg4k-bytebuddy/build.gradle

```diff
@@ -5,6 +5,7 @@ dependencies {
     compile libraries.jetbrains.kotlin.stdlib
     compile libraries.bytebuddy
 
+    testCompile libraries.jetbrains.spek.api
```

Comment:
Here and below: it used to be inherited from `core` project as a `compile` dependency.

Reply:
I don´t know how I arrived to it but thanks for noticing